app/mailers/application_mailer.rb

```diff
@@ -13,7 +13,7 @@ def user_email_with_name(user)
     email_address_with_name(user.email, name)
   end
 
-  def mail_to_user(user, subject)
-    mail(to: user_email_with_name(user), subject:)
+  def mail_to_user(user, subject, **options)
+    mail(to: user_email_with_name(user), subject:, **options)
```

Comment:
I needed the kwargs argument to allow me to override the `reply_to` address in the mailshots mailer.